bin/ch-builder2tar

```diff
@@ -128,6 +128,32 @@ EOF
     tar rf "$tar" -b1 -P --xform="s|${temp}|ch/environment|" "$temp"
     rm "$temp"
 
+    echo "adding metadata"
+    temp=$(mktemp --tmpdir ch-builder2tar.XXXXXX)
+    buildah inspect --format='{{print "{\n"}}'\
```

Comment:
Use indentation to clarify the code structure here. You could also put the format in a variable if that helps.

Reply:
Not able to indent code.